htdocs/commande/stats/index.php

```diff
@@ -46,7 +46,9 @@
 	accessforbidden();
 }
 
-$object_status = GETPOST('object_status', 'intcomma');
+$object_status = GETPOST('object_status', 'array');
```

Comment:
object_status does not seem to be an array.
I do not experience trouble on page of stats of order, even when using the filter on status

Reply:
Done @eldy